internal_ws/ykbh/src/lib.rs

```diff
@@ -53,7 +53,7 @@ impl LocalMem {
                     }
                     _ => todo!(),
                 },
-                ConstantInt::SignedInt(_si) => todo!(),
```

Comment:
Whilst I agree with this change this time, it can be useful to name unused fields, especially if there are lots of them.